### DISCUSSION TITLE
New border and text tokens

### DISCUSSION THREAD

**Discussion by HectorSancho** [Telefonica/mistica-design#1695] [Proposals]
### Goal

Detach the control and activated control tokens and create two new tokens to better control the style in components such as chips and text fields.

Tokens proposal:

- textActivated
- borderActivated

![Captura de pantalla 2024-03-08 a las 11 59 43](https://github.com/Telefonica/mistica-design/assets/56122343/bbf49a38-4956-4508-9906-b55e48b246ea)

### Context

In the current definition we do not have much control over the color of these tokens, as they have dependencies with other components. It especially impacts on dark modes

### Impact

Improve color definition in dark modes and be able to detach color styles from chips and other components of control tokens

### Discussion Questions

is it necessary?

**Comment by HectorSancho** [comment]
New tokens proposal:

borderSelected (already exists)
textSelected
textError

maybe we should add borderSelectedError? (to indicate when a text field has an error)

**Reply by yceballost** [reply]
`borderError` maybe?

**Comment by yceballost** [comment]
Created a ticket to work on it #1733
